.gitignore

```diff
@@ -9,3 +9,7 @@
 node_modules/
 vendor/
 .terraform/
```

Comment:
Do you want `.terraform/` in .gitignore even though there's no Terraform config in this repo?

Reply:
Ok.  Why didn't you mention .idea/, node_modules/, and vendor/?  I don't have those either.

Reply:
Good question. I don't know!